z3rro/src/prover.rs

```diff
@@ -152,36 +243,76 @@ impl<'ctx> Prover<'ctx> {
         self.add_assumption(&value.not());
         self.min_level_with_provables.get_or_insert(self.level);
     }
-
+    
```

Comment:
Be sure to have your editor automatically remove trailing spaces. `cargo fmt --all` should remove these as well, I think.